README.md

```diff
@@ -1,2 +1,57 @@
-# web-replay-gen
-Static Site Generator for Viewing Web Archives (in WACZ) format
+# Web Replay Gen
+
+Generate a website for viewing web archive collections with minimal setup.
+
+Compatible with web archives in [WACZ format](https://specs.webrecorder.net/wacz/latest/).
+
+**Generated website features:**
+
+- Index page listing web archives
+- Page for replaying individual web archives
+  <!-- - Automatic sitemap generation -->
+  <!-- - IPFS support -->
+
+## Getting Started
+
+### 1. Clone `web-replay-gen`
+
+```
+git clone --depth=1 --branch=main https://github.com/webrecorder/web-replay-gen.git my-web-archive-site
```

Comment:
Do we want to allow cloning any branch, or just main for simpliciy?

Also, marked this as a template repository, which means there's now a 'Use this Template' button which I guess creates a copy but not a fork for the repo. Could also add that info about that later.